pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
```

Comment:
What would you consider to be the getting-started / init command? Or the simplest one? These are better candidates for the first command than the most frequent one, accordign to [guideline n.4](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).